readyplayerme/Unity-Core
# Small fix to prevent installer from running everytime Unity recompiles

## [TICKETID](https://ready-player-me.monday.com/boards/2563815861/pulses/TICKETID)

## Description

-   previously the module installer would check and install missing modules everytime you recompile. With this update it will now check only once per session. So if you remove a module, EG glTFast it won't auto reinstall, until you reopen Unity. 




## Changes

#### Updated

-   Module installer now has an Init method that is only called once per session via EntryPoint.cs
- also moved the `#if !DISABLE_AUTO_INSTALLER`, instead of being wrapped around the whole moduleInstaller class it's now only wrapped around ModuleInstalelr.Init() inside Entry point. This should also help with previous issues of ModuleInstaller being missing inside Unity-SDK project. 



## How to Test

-   OPen a project, import core module https://github.com/readyplayerme/Unity-Core.git#feature/installer-fix
- then in editor remove glTFast, and check that it doesn't auto reimport until you restart Unity. 



## Checklist


## QA Testing

-   Mention any useful test cases for this feature and add to QA Testing documentation.

## Details

-   If more details are necessary to support the description with images and videos add them here.
